### DISCUSSION TITLE
Why we don't have TCP_QUICKACK option in TCP Stream?

### DISCUSSION THREAD

**Discussion by fy2462** [tokio-rs/tokio#4787] [Q&A]
Hi guys:

Does anybody know why I can not find the `set_quickack` function in TCP Stream?

I want to have lower latency ASAP in TCP stream.

Thanks.

**Comment by Noah-Kennedy** [comment]
You can set this using this method on either `socket2::Socket` or `socket2::SockRef`: https://docs.rs/socket2/latest/socket2/struct.Socket.html#method.quickack
